src/Mod/Draft/DraftEdit.py

```diff
@@ -466,13 +497,13 @@ def startEditing(self, event):
         if ep is None:
             return
 
-        doc = FreeCAD.getDocument(str(node.documentName.getValue()))
+        doc = App.getDocument(str(node.documentName.getValue()))
         self.obj = doc.getObject(str(node.objectName.getValue()))
         if self.obj is None:
             return
         self.setPlacement(self.obj)
 
-        FreeCAD.Console.PrintMessage(str(self.obj.Name)
+        App.Console.PrintMessage(str(self.obj.Name)
                                      + str(": editing node: n° ")
```

Comment:
Just a note here. I am not a fan of using the symbol `n°`. I would prefer to use the word spelt out, like
```py
PrintMessage(obj.Name + "editing node number:" + str(ep) + "\n")
```

I think it's better for translation.

Also, I don't think it makes sense to pass the `Name` of the object, and the string inside the `str()` functions. These are already strings.